browser/brave_prefs_browsertest.cc

```diff
@@ -47,7 +47,7 @@
 #endif
 
 #if BUILDFLAG(ENABLE_BRAVE_VPN) && !BUILDFLAG(IS_ANDROID)
```

Comment:
!BUILDFLAG(IS_ANDROID) should be dropped here

Reply:
done